build.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const fs = require('fs');
+const http = require('follow-redirects').http;
```

Comment:
I think we won't need `follow-redirects`, or do we?

Reply:
I get the content of a redirect page when I do it without the follow-redirects

Reply:
Maybe because of the `https` instead of `http`